vector/config.toml

```diff
@@ -2,13 +2,17 @@ data_dir = "./"
 
 [sources.logs]
     type = "file"
-    include = ["../bench-data/*.nljson"]
+    include = ["./bench-data/*.nljson"]
     data_dir = './'
     fingerprinting.strategy = "device_and_inode"
 
-[transforms.logs_parse_json]
-    inputs = [ "logs" ]
-    type = "json_parser"
+[transforms.json_parser]
```

Comment:
`type = "json_parse"` removes in [vector v0.12](https://vector.dev/highlights/2021-02-16-0-12-upgrade-guide/)